tests/console/rmt/rmt_feature.pm

```diff
@@ -90,7 +100,17 @@ sub run {
         die 'cleanup repos failed';
     }
 
+    record_info('add custom repo');
+    assert_script_run("rmt-cli repos custom add https://download.opensuse.org/repositories/games:/tools/SLE_15_SP3/x86_64/ Games");
+    assert_script_run("rmt-cli repos custom list | grep Games");
+    # attach the custom repo to a product
+    assert_script_run("rmt-cli products enable sle-module-development-tools/15.3/x86_64");
```

Comment:
please use setting to avoid hard code 15.3

Reply:
> In the 'record_info' the first character be upper

OK, update the record_info content, thanks.

Reply:
Modified the code here to avoid hard code, thanks.